doc/lsst.analysis.tools/troubleshooting.rst

```diff
@@ -0,0 +1,50 @@
+.. _analysis-tools-troubleshooting:
+
+Hints, Tips and Troubleshooting
+===============================
+
+What Order are Configs Applied In?
+----------------------------------
+
+In analysis tools there are a number of ways to set config options which are applied in a certain order. If 
+you get this order wrong you might be suprised by what your action is doing. In order from first applied to
+last applied:
+
+1. The action defaults
+2. The config file in the obs package
+3. The camera specific config in the obs package
+4. The command line --config and --config-file options
+5. The pipeline file configs
```

Comment:
My understanding is that 4 and 5 should be switched here. I.e., anything you give on the command line at `pipetask run` time will _always_ take precedence.

I think this block may be missing a few options as well. Let me ping @natelust to weigh in.